NuRadioReco/detector/RNO_G/RNO_single_station.json

```diff
@@ -0,0 +1,380 @@
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 2,
+      "station_id": 11
+    },
+    "3": {
+      "ant_comment": "phased array",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -94.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 3,
+      "station_id": 11
+    },
+    "4": {
+      "ant_comment": "power string Hpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -93.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "RNOG_quadslot_finalMatch",
+      "channel_id": 4,
+      "station_id": 11
+    },
+    "5": {
+      "ant_comment": "power string Hpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -92.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "RNOG_quadslot_finalMatch",
+      "channel_id": 5,
+      "station_id": 11
+    },
+    "6": {
+      "ant_comment": "power string Vpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -80.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 6,
+      "station_id": 11
+    },
+    "7": {
+      "ant_comment": "power string Vpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -60.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 7,
+      "station_id": 11
+    },
+    "8": {
+      "ant_comment": "power string Vpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -40.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
+      "channel_id": 8,
+      "station_id": 11
+    },
+    "9": {
+      "ant_comment": "Helper string B",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": -17.3,
+      "ant_position_y": -10.0,
+      "ant_position_z": -96.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
```

Comment:
do you have custom local antenna models? shouldn't it be `RNOG_vpol_4inch_center_n1.73`

Reply:
I think Ilse renamed it when she uploaded it to the server.